elastic/beats
# Added support for couchdb v3.

## What does this PR do?

Added Metricbeat support for CouchDB version 3. It seems like the result format of the couchdb stats endpoints did not change from version 2 to 3 (the release notes do not mentioned changes to the stats endpoints).
So I'm just reusing the stats format of version 2.

## Why is it important?


Added support for the lastest major CouchDB version 3.

## Checklist



- [x ] My code follows the style guidelines of this project
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## How to test this PR locally


- Install CouchDB version 3.x (https://couchdb.apache.org/)
- Configure the CouchDB module for Metricbeat. (https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-module-couchdb.html)
- Check that the Kibana dashboard "[Metricbeat CouchDB] Database Overview" is filled with data.

## Related issues



- Closes #22743
